rollbar/contrib/django/middleware.py

```diff
@@ -18,11 +18,16 @@
 
 import rollbar
 
+import django
 from django.core.exceptions import MiddlewareNotUsed
-from django.core.urlresolvers import resolve
 from django.conf import settings
 from django.http import Http404
 
+if django.VERSION >= (1, 10):
```

Comment:
I'm happy to change it but checking the version is a pretty common pattern:

DRF: https://github.com/encode/django-rest-framework/blob/5ba2368ff9b64bef6eced25985bc2fc5bbd08a90/rest_framework/compat.py#L130-L151